conda_forge_tick/migrators/pip_wheel_dep.py

```diff
@@ -0,0 +1,152 @@
+import tempfile
+from typing import Any
+import typing
+
+import requests
+
+from conda_forge_tick.migrators import MiniMigrator
+from conda_forge_tick.xonsh_utils import indir
+
+if typing.TYPE_CHECKING:
+    from conda_forge_tick.migrators_types import AttrsTypedDict
+
+
+class PipWheelMigrator(MiniMigrator):
+    post_migration = True
+
+    def filter(self, attrs: "AttrsTypedDict", not_bad_str_start: str = "") -> bool:
+        run_reqs = attrs.get("requirements", {}).get("run", set())
+        source_url = attrs.get("url") or attrs.get("source", {}).get("url")
+        url_names = ["pypi.python.org", "pypi.org", "pypi.io"]
+        if not any(s in source_url for s in url_names):
+            return True
+        if (
+            not attrs.get("meta_yaml", {})
+            .get("extra", {})
+            .get("bot", {})
+            .get("run_deps_from_wheel", False)
+        ):
```

Comment:
This should use `conda-forge.yml`'s `bot` setting instead of `meta.yaml`

Reply:
yeah thats probably a better place for it